From 5ad9e8b26c48b456251175eb211fd52f6b8d207c Mon Sep 17 00:00:00 2001
From: Thomas Haller <thaller@redhat.com>
Date: Fri, 8 Dec 2017 14:20:35 +0100
Subject: [PATCH 1/1] Revert "systemd: let "NetworkManager-wait-online.service"
 require "NetworkManager.service""

Upstream changes "NetworkManager-wait-online.service" to require NM.

See
 - https://cgit.freedesktop.org/NetworkManager/NetworkManager/commit/?id=28b97f02f64ecf94e22c8929dfd0ba64151bd9d3
 - https://bugzilla.redhat.com/show_bug.cgi?id=1452866#c50
 - https://github.com/systemd/systemd/commit/9db307820e6f545665fc87f255af737228b7183c

This is a change in behavior, that we want to avoid for RHEL.
See also Fedora bug https://bugzilla.redhat.com/show_bug.cgi?id=1513613

This reverts commit 28b97f02f64ecf94e22c8929dfd0ba64151bd9d3.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1520865
---
 data/NetworkManager-wait-online.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/NetworkManager-wait-online.service.in b/data/NetworkManager-wait-online.service.in
index 896324685..1753d2039 100644
--- a/data/NetworkManager-wait-online.service.in
+++ b/data/NetworkManager-wait-online.service.in
@@ -1,7 +1,7 @@
 [Unit]
 Description=Network Manager Wait Online
 Documentation=man:nm-online(1)
-Requires=NetworkManager.service
+Requisite=NetworkManager.service
 After=NetworkManager.service
 Before=network-online.target
 
-- 
2.14.3

